src/Relation/Binary/Consequences.agda

```diff
@@ -164,3 +164,9 @@ module _ {P : REL A B ℓ₁} {Q : REL B A ℓ₂} where
 
   flip-Connex : Connex P Q → Connex Q P
   flip-Connex f x y = Sum.swap (f y x)
+
+module _ {a b r} {A : Set a} {B : Set b} {R : REL A B r} where
+     dec⟶irrel′ : Decidable R → Irrelevant′ R
+     dec⟶irrel′ dec {a} {b} _ with dec a b
+     dec⟶irrel′ _ _ | yes p = p
+     dec⟶irrel′ _ r | no ¬p = ⊥-elim (¬p r)
```

Comment:
`dec⟶irrel′` could be implemented using [`recompute`](https://agda.github.io/agda-stdlib/Relation.Nullary.html#782), couldn't it?